src/index.js

```diff
@@ -0,0 +1,265 @@
+  if (clicked.classList.contains('seeBtn')) {
+    makeTaskSecOnClck(currentProject, selectedTasks);
+  }
+});
+
+first.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const clicked = e.target;
+  if (clicked.classList.contains('plus')) {
+    makeTaskForm();
+  }
+});
+
+const taskForm = document.getElementById('taskForm');
+taskForm.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const clicked = e.target;
+
+  if (clicked.classList.contains('dismissTask')) {
+    taskForm.innerHTML = '';
+    countForm = false;
+    return;
+  }
+
+  if (clicked.classList.contains('confirmTask')) {
+    const newTitle = taskForm[0].value;
+    const newComment = taskForm[1].value;
+    const newPriority = taskForm[2].value;
+    let newDate = taskForm[3].value;
+
+    let confirm = true;
+
+    if (!newTitle) {
+      alert('Title must not be empty');
+      confirm = false;
+      return;
+    }
+
+    if (confirm === false) {
+      // eslint-disable-next-line
+      return false;
+    }
+
+    if (!newDate) {
+      newDate = setDate();
+    }
+
+    const newTask = new Task(newTitle, currentProject, newComment, newPriority, newDate);
+
+    myTasks.push(newTask);
+    setTasks();
+    const selectedTasks = myTasks.filter((task) => task.projectsN === currentProject);
+
+    makeTaskSection(currentProject, selectedTasks);
+    makeToday();
+    taskForm.reset();
+  }
+});
+
+tableArticle.addEventListener('click', (e) => {
+  e.preventDefault();
+  const clicked = e.target;
+  // eslint-disable-next-line
+  const index = parseInt(clicked.closest('tr').firstElementChild.textContent) - 1;
+  console.log('this', index);
+
+  if (clicked.classList.contains('deleteTaskBtn')) {
+    alert('Are you sure?');
+    clicked.closest('tr').remove();
+    myTasks.splice(index, 1);
+    setTasks();
+    makeToday();
+  }
+
+  if (clicked.classList.contains('statusBTN')) {
+    changeStatus(index);
+    setTasks();
+    const selectedTasks = myTasks.filter((task) => task.projectsN === currentProject);
+    makeTaskSection(currentProject, selectedTasks);
+    makeToday();
+  }
+
+  if (clicked.classList.contains('editTaskBtn')) {
+    const taskToEdit = myTasks[index];
+    makeEditForm(taskToEdit);
+    editTaskForm.addEventListener('click', (e) => {
+      e.preventDefault();
+      const clicked = e.target;
+      if (clicked.classList.contains('dismissEditTask')) {
+        editTaskForm.innerHTML = '';
+      }
+      if (clicked.classList.contains('confirmEditTask')) {
+        // eslint-disable-next-line
+
+        const editTitle = editTaskForm[0].value;
+        const editComment = editTaskForm[1].value;
+        const editPriority = editTaskForm[2].value;
+        let editDate = editTaskForm[3].value;
+
+        let confirm = true;
+
+        if (!editTitle) {
+          alert('Title must not be empty');
+          confirm = false;
+          return;
+        }
+
+        if (confirm === false) {
+          // eslint-disable-next-line
```

Comment:
Kindly avoid the ESLInt inline disabling rule, it is the same as altering the ESLint config files